### PR TITLE
Draft thank you blog post 2021

### PR DESCRIPTION
**Pull request by steffilazerte**
Hi Stef, here's the draft post. I've filled in the numbers where I could and indicated with XXX or notes where I don't have the numbers at all or yet. It's a tad on the boring side (I considered references the pets again but Alex thought it was forced so I let it go. Let me know if you want any changes, etc.).

### CODE REVIEW

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+We often *think* our thank yous, but *expressing* thank yous can be incredibly meaningful.
+Try to be specific. 
+What did they do? 
+What impact did their actions have?
+
+On that note, we also want to take a moment to say **thank you**, to YOU.
+Thank you so much for making the rOpenSci community that much more awesome.
+
+Although it's been another hard year across the globe, this community has continued to come together to build good things.
+We've had 20 packages successfully complete [software peer review](/software-review/)[^1] with the help of ~40 [reviewers](https://devguide.ropensci.org/softwarereviewintro.html#reviewers) and 11 [editors](/about/#team), including 3 [new editors](/blog/2021/10/12/editors2021/).
```
Comment (maelle) [inline content/blog/2021-12-16-thank-you-2021/index.md:32]:
```suggestion
We've had 20 packages successfully complete [software peer review](/software-review/)[^1] with the help of about 40 [reviewers](https://devguide.ropensci.org/softwarereviewintro.html#reviewers) and 11 [editors](/about/#team), including 3 [new editors](/blog/2021/10/12/editors2021/). 
```

The tilde looks weird with the website font (too high).

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+What did they do? 
+What impact did their actions have?
+
+On that note, we also want to take a moment to say **thank you**, to YOU.
+Thank you so much for making the rOpenSci community that much more awesome.
+
+Although it's been another hard year across the globe, this community has continued to come together to build good things.
+We've had 20 packages successfully complete [software peer review](/software-review/)[^1] with the help of ~40 [reviewers](https://devguide.ropensci.org/softwarereviewintro.html#reviewers) and 11 [editors](/about/#team), including 3 [new editors](/blog/2021/10/12/editors2021/). 
+Many people contributed code or documentation. Similar to last year, 85 people made their first code contribution to rOpenSci this year! 
+Others shared [use cases](/usecases/) (88 new ones!) to help package authors see how their work is being used and help other users imagine how they can apply it.
```
Comment (maelle) [inline content/blog/2021-12-16-thank-you-2021/index.md:34]:
```suggestion
Others shared 88 new [use cases](/usecases/) to help package authors see how their work is being used and help other users imagine how they can apply it.
```

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Our community has benefited from the voices of many different people this year. Our [blog](/archive/) had 34 community member authors, including 13 first-time authors, [Community Calls](/commcalls/) were attended by over 200 people in 27 countries on 5 continents, and virtual [social coworking and office hours](/blog/2021/08/17/coworking-sessions/) brought together people from at least 12 countries to get their own work done and to join in some spirited scavenger hunts!
+Many others have continued to cite our software, ask or answer questions, open issues to report bugs or request features, make pull requests to fix things, weigh in on standards and best practices, invited us to talk about our work, or mentioned us in a presentation, a post, or on Twitter.
+
+Some people’s volunteer contributions this year warrant a special mention (alpha-sorted by first name):
```
Comment (maelle) [inline content/blog/2021-12-16-thank-you-2021/index.md:39]:
```suggestion
Some people’s volunteer contributions this year warrant a special mention (alphabetically sorted by first name):
```

The abbreviation was new to me so I stumbled on it. :wolf:

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+*   Editors for [Software Peer Review](/software-review/), [Adam Sparks](/author/adam-sparks/), [Anna Krystalli](/author/anna-krystalli/), [Emily Riederer](/emily-riederer/), [Laura DeCicco](/author/laura-decicco/), [Jeff Hollister](/author/jeff-hollister/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Melina Vidoni](/author/melina-vidoni/), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/);
```
Comment (maelle) [inline content/blog/2021-12-16-thank-you-2021/index.md:41]: Brooke Anderson was editor until February.

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+*   New maintainers of existing rOpenSci packages, [Alban Sagouis](https://github.com/AlbanSagouis), [David Selby](https://selbydavid.com/), [Hannah Owens](https://github.com/hannahlowens), [John Waller](https://github.com/jhnwllr), [Lluís Revilla Sancho](https://llrs.dev/), [Salix Dubois](https://github.com/salix-d); 
+*   Presenters in our Community Calls, [Alex Hayes](/author/alex-hayes/), [Anna Krystalli](/author/anna-krystalli/), [Arfon Smith](/author/arfon-smith/), [Christoph Sax](/author/christoph-sax/), [Hugo Gruson](/author/hugo-gruson/), [John Sakaluk](/author/john-sakaluk/), [Juliane Manitz](/author/juliane-manitz/), [Rebecca Killick](/author/rebecca-killick/), and the many many people who commented in our shared docs;
+*   Contributors to our guides
+    *   [rOpenSci Statistical Software Peer Review](https://ropenscilabs.github.io/statistical-software-review-book/index.html), [Christoph Sax](/author/christoph-sax/), [Jakub Nowosad](https://github.com/Nowosad), [Jem Stachelek](https://github.com/jsta), and stats peer review Editors;
```
Comment (maelle) [inline content/blog/2021-12-16-thank-you-2021/index.md:46]:
```suggestion
    *   [rOpenSci Statistical Software Peer Review](https://ropenscilabs.github.io/statistical-software-review-book/index.html): [Christoph Sax](/author/christoph-sax/), [Jakub Nowosad](https://github.com/Nowosad), [Jem Stachelek](https://github.com/jsta), and stats peer review Editors;
```

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    *   [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/), [Frans van Dunné](https://github.com/FvD), [Hugo Gruson](/author/hugo-gruson/), and peer review Editors;
```
Comment (maelle) [inline content/blog/2021-12-16-thank-you-2021/index.md:47]:
```suggestion
    *   [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/): [Frans van Dunné](https://github.com/FvD), [Hugo Gruson](/author/hugo-gruson/), and peer review Editors;
```

Comment (maelle) [inline content/blog/2021-12-16-thank-you-2021/index.md:47]: Also (small edits that I apparently forgot to list in release posts :grimacing: )

* Christophe Dervieux https://github.com/ropensci/dev_guide/pull/346
* Lluís https://github.com/ropensci/dev_guide/pull/306

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    *   [rOpenSci Community Contributing Guide](https://contributing.ropensci.org/), [Carlos Cámara](https://www.carloscamara.es/), [Jonas Hagenberg](https://github.com/jonas-hag), [Leon Lenostatos](https://github.com/Lenostatos);
+*   External member of our Code of Conduct Committee, [Kara Woo](/author/kara-woo/);
+*   rOpenSci Strategic Advisors, [Carl Boettiger](/author/carl-boettiger/), [Jenny Bryan](/author/jenny-bryan/), [Rich FitzJohn](/author/rich-fitzjohn/).
+
+Thank you **all** for making rOpenSci.
```
Comment (maelle) [inline content/blog/2021-12-16-thank-you-2021/index.md:52]: Isn't the post lacking a conclusion?

**Review by maelle — COMMENTED** [review]
The existence of this post makes me so happy, and I really like the message at the beginning! Few comments, see also my slack comments regarding the new maintainers list.

File: content/blog/2021-12-16-thank-you-2021/index.md (hunk 2 above)
Comment (maelle) [reply content/blog/2021-12-16-thank-you-2021/index.md:34]: or any phrasing tweak that'd avoid the parentheses.

File: content/blog/2021-12-16-thank-you-2021/index.md (hunk 4 above)
Comment (stefaniebutland) [reply content/blog/2021-12-16-thank-you-2021/index.md:41]:
```suggestion
*   Editors for [Software Peer Review](/software-review/), [Adam Sparks](/author/adam-sparks/), [Anna Krystalli](/author/anna-krystalli/), [Emily Riederer](/emily-riederer/), [Laura DeCicco](/author/laura-decicco/), [Jeff Hollister](/author/jeff-hollister/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Melina Vidoni](/author/melina-vidoni/), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/) and alumnus [Brooke Anderson](/author/brooke-anderson/);
```

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+Some people’s volunteer contributions this year warrant a special mention (alpha-sorted by first name):
+
+*   Editors for [Software Peer Review](/software-review/), [Adam Sparks](/author/adam-sparks/), [Anna Krystalli](/author/anna-krystalli/), [Emily Riederer](/emily-riederer/), [Laura DeCicco](/author/laura-decicco/), [Jeff Hollister](/author/jeff-hollister/), [Julia Gustavsen](/author/julia-gustavsen/), [Mauro Lepore](/author/mauro-lepore/), [Melina Vidoni](/author/melina-vidoni/), plus rOpenSci team members who volunteer as Editors, [Karthik Ram](/author/karthik-ram/), [Noam Ross](/author/noam-ross/), [Maëlle Salmon](/author/maëlle-salmon/);
+*   Advisory Board and Editors for [Statistical Software Peer Review](/stat-software-review/), [Ben Bolker](https://ms.mcmaster.ca/~bolker/), [Leonardo Collado-Torres](/author/leonardo-collado-torres/), [Paula Moraga](http://www.paulamoraga.com/), [Rebecca Killick](/author/rebecca-killick/), [Stephanie Hicks](https://www.stephaniehicks.com/), [Toby Hocking](https://tdhock.github.io/), and alumni [Martin Morgan](https://www.roswellpark.org/martin-morgan), [Max Kuhn](https://appliedpredictivemodeling.com/);
```
Comment (stefaniebutland) [inline content/blog/2021-12-16-thank-you-2021/index.md:42]:
```suggestion
*   Advisory Board and Editors for [Statistical Software Peer Review](/stat-software-review/): [Ben Bolker](https://ms.mcmaster.ca/~bolker/), [Leonardo Collado-Torres](/author/leonardo-collado-torres/), [Paula Moraga](http://www.paulamoraga.com/), [Rebecca Killick](/author/rebecca-killick/), [Stephanie Hicks](https://www.stephaniehicks.com/), [Toby Hocking](https://tdhock.github.io/), and alumni [Martin Morgan](https://www.roswellpark.org/martin-morgan), [Max Kuhn](https://appliedpredictivemodeling.com/);
```

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+*   New maintainers of existing rOpenSci packages, [Alban Sagouis](https://github.com/AlbanSagouis), [David Selby](https://selbydavid.com/), [Hannah Owens](https://github.com/hannahlowens), [John Waller](https://github.com/jhnwllr), [Lluís Revilla Sancho](https://llrs.dev/), [Salix Dubois](https://github.com/salix-d);
```
Comment (stefaniebutland) [inline content/blog/2021-12-16-thank-you-2021/index.md:43]:
```suggestion
*   New maintainers of existing rOpenSci packages: [Alban Sagouis](https://github.com/AlbanSagouis), [David Selby](https://selbydavid.com/), [Hannah Owens](https://github.com/hannahlowens), [John Waller](https://github.com/jhnwllr), [Lluís Revilla Sancho](https://llrs.dev/), [Salix Dubois](https://github.com/salix-d), [Sharla Gelfand](https://www.sharlagelfand.com/), [Brenton Wiernik](https://wiernik.org/); 
```

File: content/blog/2021-12-16-thank-you-2021/index.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+*   New maintainers of existing rOpenSci packages, [Alban Sagouis](https://github.com/AlbanSagouis), [David Selby](https://selbydavid.com/), [Hannah Owens](https://github.com/hannahlowens), [John Waller](https://github.com/jhnwllr), [Lluís Revilla Sancho](https://llrs.dev/), [Salix Dubois](https://github.com/salix-d); 
+*   Presenters in our Community Calls, [Alex Hayes](/author/alex-hayes/), [Anna Krystalli](/author/anna-krystalli/), [Arfon Smith](/author/arfon-smith/), [Christoph Sax](/author/christoph-sax/), [Hugo Gruson](/author/hugo-gruson/), [John Sakaluk](/author/john-sakaluk/), [Juliane Manitz](/author/juliane-manitz/), [Rebecca Killick](/author/rebecca-killick/), and the many many people who commented in our shared docs;
```
Comment (stefaniebutland) [inline content/blog/2021-12-16-thank-you-2021/index.md:44]:
```suggestion
*   Presenters in our Community Calls: [Alex Hayes](/author/alex-hayes/), [Anna Krystalli](/author/anna-krystalli/), [Arfon Smith](/author/arfon-smith/), [Christoph Sax](/author/christoph-sax/), [Hugo Gruson](/author/hugo-gruson/), [John Sakaluk](/author/john-sakaluk/), [Juliane Manitz](/author/juliane-manitz/), [Rebecca Killick](/author/rebecca-killick/), and the many many people who commented in our shared docs;
```

**Review by stefaniebutland — APPROVED** [review]

**Pull request merged**